nipype/algorithms/misc.py

```diff
@@ -317,51 +331,26 @@ def _list_outputs(self):
         return outputs
 
 
-class GzipInputSpec(GunzipInputSpec):
-    mode = traits.Enum("compress", "decompress", usedefault=True)
+class GunzipInputSpec(GzipInputSpec):
+    mode = traits.Enum("decompress", "compress", usedefault=True,
```

Comment:
This change would make `gunzip` decompress-only again by disabling `compress` mode.

```suggestion
    mode = traits.Enum("decompress", usedefault=True,
```